Assets/MixedRealityToolkit-Preview/LightEstimation-Examples/EditorGyro/FollowGyroscope.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Preview.LightEstimation
```

Comment:
namespace should have "Examples" somewhere (like in the demo script)

Reply:
Shifted a few files around here, but I did review the namespaces, and they should be fine now :)